pkgs/development/python-modules/httplib2/default.nix

```diff
@@ -25,14 +25,6 @@ buildPythonPackage rec {
     sha256 = "sha256-1zqs3YRVtm5DwewETLtRg5XhMJPJsMi0QLfeGirOURs=";
   };
 
-  patches = [
-    # fix test_inject_space
-    (fetchpatch {
```

Comment:
Can you please remove the [`fetchpatch` input](https://github.com/NixOS/nixpkgs/pull/143809/files#diff-5a3017243779e62f1890d1e25a232aef019a1af868e5e526d3725f3296b4b21bL5)?

Reply:
Oh, good catch! Always get it missed. Updated.